usr/local/sbin/resticw

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+# Exit on error, unbound variable, pipe error
+set -euo pipefail
+
+DOC="A little wrapper over restic just to handle profiles and environment loading.
+
+  It loads the backup profile/environment in a subshell to avoid any credential leak (Note: Run it with sudo so it can load the environment).
+
+  Usage:
+    resticw [options] <restic_arguments>
```

Comment:
When just running like `$ resticw` without arguments, the helptext is missing the 2 last characters from this line:

```
   Usage:
      resticw [options] <restic_arguemnt
                                        ^
```

Reply:
I think restic calls the first level of argument commands, which can have arguments. In `$restic backup /` `backup` would be the command and `/` the argument to that command =>

```
 Usage:
    resticw [restic_options] <restic_command> [restic_command_options]
```

?

Reply:
Yep! I realised the missing text on `$ resticw` before, and after comparing with the docopt.sh example, now, I managed to fix it. It should work if you update the branch :)

---
For the other thing of `<restic_command> [restic_command_options]` I tried to change it and just finally reworded as `<restic_arguments_line>...`. I think splitting in 2 (command, options) does not work well with the parser. But it help me to fix something, now there is `...` that was missing to allow multiple tokens. Also, from the shell standpoint, they are the arguments provided to the restic command (it depends on which perspective you observe, hehe).

But the ultimate reason is that it didn't work well.

Reply:
Now it looks better :)